pkgs/development/compilers/crystal/default.nix

```diff
@@ -91,39 +93,44 @@ let
     ];
 
     FLAGS = [
-      "--release"
+      # "--release"
```

Comment:
Is that change intentional? If so, why? :)

Reply:
Good catch :)

Reply:
Had that there to speed up compiles during testing.